packages/core/src/input/BasicInput.tsx

```diff
@@ -8,6 +8,7 @@ import React, {
 
 import {MaskInput, Input, Value} from '../primitive'
 import {MaskedInput, isMaskDigital, Mask, Pipe} from '../mask'
+import {useTheme} from '@emotion/react'
```

Comment:
It should be wrapped somehow. Like https://github.com/qiwi/pijma/blob/master/packages/core/src/styled.ts